.github/workflows/coverage.yml

```diff
@@ -30,28 +30,15 @@ jobs:
       - name: Install rustfilt symbol demangler
         run: |
           cargo install rustfilt
-      - name: Rerun tests for coverage
+
+      - name: Install cargo-llvm-cov cargo command
+        run: |
+          cargo install cargo-llvm-cov --version 0.1.0-alpha.3
```

Comment:
Why pin a specific alpha version here?
Pinned versions tend to cause CI failures when they get outdated.

(If there's a specific reason, let's add a comment and a TODO for stable.)

Reply:
On crates.io, when only pre-release versions have been published, it only allows you to publish via the explicit version tag, but we can just do `1.0.0-alpha`:

![image](https://user-images.githubusercontent.com/552961/121938288-9c97c280-cd19-11eb-9768-bdce9d721cdf.png)

![image](https://user-images.githubusercontent.com/552961/121938422-ce108e00-cd19-11eb-9b7e-59ab38cc54b8.png)

Reply:
Ha, that didn't work:

![image](https://user-images.githubusercontent.com/552961/121939406-006ebb00-cd1b-11eb-8696-3d2cee174e2a.png)

Reply:
I think you might have wanted `0.1.0-alpha.0` or `^0.1.0-alpha.0` or `~0.1.0-alpha.0`:
https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#caret-requirements